.pr_agent.toml

```diff
@@ -0,0 +1,39 @@
+[config]
+model="gpt-4" # the OpenAI model you've deployed on Azure (e.g. gpt-3.5-turbo)
+model_turbo="gpt-4" # the OpenAI model you've deployed on Azure (e.g. gpt-3.5-turbo)
+fallback_models=["gpt-4"] # the OpenAI model you've deployed on Azure (e.g. gpt-3.5-turbo)
+
+[github_action_config]
+auto_review = true
+auto_describe = true
+auto_improve = true
+pr_actions = ["opened", "reopened", "ready_for_review", "review_requested"]
+
+[pr_reviewer] # /review #
+# enable/disable features
+require_score_review=false
+require_tests_review=true
+require_estimate_effort_to_review=true
+require_can_be_split_review=false
+require_security_review=true
+require_ticket_analysis_review=true
+# general options
+num_code_suggestions=0
```

Comment:
I'm curious about some of these settings - it might be helpful to either link out to docs or add comments explaining why non-obvious things are set certain ways (like this PR Agent actually suggests having a non-0 number for this setting, but I don't entirely understand what it does)

Reply:
That's a fair point. I'd have to see where they have docs for these settings because I only found a link to their example settings and not an explanation of the settings themselves.